### PR TITLE
feat: add first-time.yml action

### PR DESCRIPTION
**Pull request by thuongtruong1009**
## Checklist
 
- [x ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x ] My commit messages follow [best practices][best_practices].
- [x ] My code follows the established code style of the repository.
- [x ] I added or updated tests for the changes I made (if applicable).
- [x ] I added or updated documentation (if applicable).
- [x ] I tried running the project locally and verified that there are no
  visible errors.

### CODE REVIEW

File: .github/workflows/first-time.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thanks for your first issue here! Be sure to follow the issue template! The notify email will be sent to you as soon as.'
+        pr-message: 'Thanks for your first Pull-request here! Be sure to follow the issue template! The notify email will be sent to you as soon as.'
```
Comment (brylie) [inline .github/workflows/first-time.yml]: > The notify email will be sent to you as soon as.

The above sentence is a fragment and might confuse people. So, we should consider rewording the issue/PR messages to contain complete sentences.

File: .github/workflows/first-time.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (brylie) [inline .github/workflows/first-time.yml:11]: Out of curiosity, would this possibly expose our GitHub token to the author of `actions/first-interaction`? E.g. could they possibly define a script or service that sends our GitHub token to an external location, such as by making an HTTP request, from within the `actions/first-interaction` service?

File: .github/workflows/first-time.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        
+    - name: First interaction
+      uses: actions/first-interaction@v1.1.0
+      with:
+        # Token for the repository. Can be passed in using {{ secrets.GITHUB_TOKEN }}
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        # Comment to post on an individual's first issue
+        issue-message: Thanks for your first issue here! Be sure to follow the issue template! The notify email will be sent to you as soon as.
+        # Comment to post on an individual's first pull request
+        pr-message: Thanks for your first Pull-request here! Be sure to follow the issue template! The notify email will be sent to you as soon as.
```
Comment (brylie) [inline .github/workflows/first-time.yml]: The issue and PR messages contain sentence fragments.

**Review by brylie — CHANGES_REQUESTED** [review]

File: .github/workflows/first-time.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Welcome to creativecommons community! Thanks so much for creating your first issue :)'
```
Comment (nimishbongale) [inline .github/workflows/first-time.yml:12]: We could consider rephrasing this to something along the lines of "Welcome to the Creative Commons Open Source community. Thanks for creating your first issue! 🚀 "

File: .github/workflows/first-time.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+        pr-message: 'Thanks so much for creating your first PR, the creativecommons community thanks you :)'
```
Comment (nimishbongale) [inline .github/workflows/first-time.yml:13]: We could consider rephrasing this to something along the lines of "You have successfully raised your first PR to the CC Vocabulary repository. The Creative Commons Open Source community is grateful for your contribution! 💯"

Comment (thuongtruong1009) [reply .github/workflows/first-time.yml:13]: Hi @nimishbongale @brylie 
I make a Pull Request [1059[(https://github.com/creativecommons/vocabulary/pull/1059)
Thank you very much!

**Pull request closed without merging**